### PR TITLE
cmake: fix git version string, cleanup

### PR DESCRIPTION
**Pull request by liewegas**
The version string grew a 'v' starting with the cmake transition.  Remove it, and drop the unused ceph_ver.h in the tarball.

### CODE REVIEW

**Comment by liewegas** [comment]
gitbuilder is happy at least

**Comment by liewegas** [comment]
@alimaredia ?

File: make-dist [hunk 1]
```diff
@@ -32,8 +32,9 @@ bin/git-archive-all.sh --prefix ceph-$version/ \
 		       $outfile.tar
 
 # populate files with version strings
-echo "including src/.git_version, src/ceph_ver.h, ceph.spec"
-src/make_version -g src/.git_version -c src/ceph_ver.h
+echo "including src/.git_version, ceph.spec"
+
+git rev-parse HEAD 2> /dev/null; git describe 2> /dev/null > src/.git_version
```
Comment (dang) [inline make-dist]: This is dropping the 'git rev-parse HEAD'.  The old script was capturing STDOUT, so it got both lines, but this one is only redirecting the 'git describe'

**Comment by liewegas** [comment]
D'oh, @dang fixed

**Pull request merged**